src/frontend/context/StegContext.ts

```diff
@@ -0,0 +1,29 @@
+import constate from 'constate';
+
+import { FanePath } from '../Sider/Behandling/faner';
+import { Behandling } from '../typer/behandling/behandling';
+import { erBehandlingRedigerbar } from '../typer/behandling/behandlingStatus';
+import { Steg } from '../typer/behandling/steg';
+
+const faneTilSteg: Record<FanePath, Steg> = {
+    inngangsvilkar: Steg.INNGANGSVILKÅR,
+    stonadsvilkar: Steg.VILKÅR,
+    'vedtak-og-beregning': Steg.BEREGNE_YTELSE,
+    simulering: Steg.SEND_TIL_BESLUTTER,
+    brev: Steg.SEND_TIL_BESLUTTER,
+};
+
+interface Props {
+    fane: FanePath | undefined;
+    behandling: Behandling;
+}
+
+export const [StegProvider, useSteg] = constate(({ fane, behandling }: Props) => {
+    const erStegRedigerbar = fane && behandling.steg === faneTilSteg[fane];
+    const erStegOgBehandlingRedigerbar = erStegRedigerbar && erBehandlingRedigerbar(behandling);
+
+    return {
+        erStegRedigerbar,
+        erStegOgBehandlingRedigerbar,
```

Comment:
Er det tilstrekkelig å kun returnere `erStegRedigerbar` ? 
Som då sjekker at behandlingen er redigerbar og behandlingen er i riktig steg?

Reply:
Ja det blir jo i praksis det samme. Men en behandling kan jo være i et steg selv om behandlingen ikke er redigerbar. Tenker i så fall det blir mer korrekt å kalle den `erStegOgBehandlingRedigerbar`

Reply:
Tenker egentlig at bare steg dekker behovet selv om behandlingen ikke er redigerbar. Det handler om at akkurat dette steget ikke er redigerbart nå. Så hvis man er på inngangsvilkårsiden så er ikke den redigerbar om behandlingen ikke er i steg `inngangsvilkår`. 

Uavhengig av hvilket navn som brukes så syntes jeg det burde snus på for å gjøre det lettere å lese: 
`stegOgBehandlingErRedigerbar` eller `stegErRedigerbar`  (usikker på om det er redigerbar eller redigerbart som er rett).